submissions/vl-shevchenko/dom-api-task/script.js

```diff
@@ -0,0 +1,83 @@
+const city = [
+    {
+        color: 'url(images/kyiv5-904w.jpg)',
+        name: 'Kyiv',
+        info: 'The capital of Ukraine and the most interesting city, cultural and spiritual capital of the country. Due to the hilly terrain, the town is very picturesque and has more than 20 lookouts. Please note that Kyiv is quite large and even the sightseeing tour takes 10 kilometers, so have at least 2 days for a visit.',
+    },
+    {
+        color: 'url(images/Lviv.jpg)',
+        name: 'Lviv',
+        info: 'Lvivs Old Town is a UNESCO World Heritage Site. This city is very popular among Ukrainians from other cities who often come here for the weekend. Recently Lviv turned to the restaurant capital of the country, due to a network of unusual restaurants "!FEST".',
+    },
+    {
+        color: 'url(images/Kamianets-Podilskyi.jpeg)',
+        name: 'Kamianets-Podilskyi',
+        info: 'Best small city in Ukraine. Rock towns are common for Italy or Spain, but in Ukraine it is very rare. Here is an oldest town hall in the country, a cozy old town and a fairy-tale castle.',
+    },
+    {
+        color: 'url(images/Chernivtsi.jpg)',
+        name: 'Chernivtsi',
+        info: 'Pretty unknown city, capital of the Bukovina region and cultural center of western Ukraine, along with Lviv. In 2012 was recognized the most comfortable city in the country. Known primarily for a luxurious building of the university, the former residence of the Metropolitan and UNESCO World Heritage site. Old Town is also beautiful.',
+    },
+    {
+        color: 'url(images/Uzhgorod.jpg)',
+        name: 'Uzhgorod',
+        info: 'The capital of the Carpathian Ukraine is known for cherry blossoms in May, the castle and wine festivals. There is also a lovely old town.',
+
+    },
+    {
+        color: 'url(images/Odesa.jpg)',
+        name: 'Odesa',
+        info: 'City by the Sea, the birthplace of many famous Ukrainian immigrants and the capital of humor. Here is a special atmosphere, special people and special attitude to everything, you need to feel it. In early April Odessa helds the festival of humor.',
+    },
+];
+
+let defaultContent = document.querySelector('.container');
+defaultContent.innerHTML = `<p class='info'>${city[0].info}</p>`;
+document.body.style.backgroundImage = city[0].color;
+let defaultName = document.querySelector('.name-city');
+defaultName.innerText = `${city[0].name}`;
+
+const buildMenu = city => {
+    document.querySelector('.nav__bloc').innerHTML = `<ul class='container-list'></ul>`;
+    city.map((val) => {
+        let nameCity = document.createElement('li');
+        nameCity.classList.add('menu-list');
+        nameCity.innerHTML = `<a href="#" class='menu-link'>${val.name}</a>`;
+
+        document.querySelector('.container-list').appendChild(nameCity);
```

Comment:
You already have var for `.container-list` on line [53](https://github.com/kottans/frontend-2022-homeworks/pull/426/files#diff-67c3e77f177ce6616994c08bfed32b00d13bc21085c15f08c825ebb6e7cfb2eeR53). Why you reselect it?